server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -522,30 +524,68 @@ export function configureWebSocketServices(
                         return;
                     }
 
+                    const lumberjackProperties = {
+                        [CommonProperties.clientId]: clientId,
+                        ...getLumberBaseProperties(connection.documentId, connection.tenantId),
+                    };
+                    const handleMessageBatchProcessingError = (error: any) => {
+                        if (isNetworkError(error)) {
+                            if (error.code === 413) {
+                                Lumberjack.info("Rejected too large operation(s)", lumberjackProperties);
+                                socket.emit("nack", "", [createNackMessage(
+                                    error.code,
+                                    NackErrorType.BadRequestError,
+                                    error.message,
+                                )]);
+                                return;
+                            }
+                        }
+                        Lumberjack.error("Error processing submitted op(s)", lumberjackProperties, error);
+                    };
                     messageBatches.forEach((messageBatch) => {
                         const messages = Array.isArray(messageBatch) ? messageBatch : [messageBatch];
-                        const sanitized = messages
-                            .filter((message) => {
-                                if (message.type === MessageType.RoundTrip) {
-                                    if (message.traces) {
-                                        // End of tracking. Write traces.
-                                        // TODO: add Lumber metric here?
-                                        metricLogger.writeLatencyMetric("latency", message.traces).catch(
-                                            (error) => {
-                                                logger.error(error.stack);
-                                                Lumberjack.error(error.stack);
-                                            });
+                        try {
+                            const sanitized = messages
+                                .filter((message) => {
+                                    if (message.type === MessageType.RoundTrip) {
```

Comment:
This is not a part of this PR but are we using this? Can this be removed?

Reply:
There is nowhere in the codebase using MessageType.RoundTrip other than right here. Not sure what the history is with this bit of code. Would prefer to remove it in a separate PR though